### ISSUE TITLE
NVDA 2022.3.2 install fails on Windows 10 21H2 19044 Server

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#14379]
### Steps to reproduce:
Reported by email from user

Running NVDA 2022.3.2 launcher with following switches:
`--minimal --install-silent --enable-start-on-logon=True`

### Actual behavior:

Install fails

From log:
``` 
CRITICAL - __main__ (13:36:50.390) - MainThread (2316):
core failure
ValueError: -1 is not a valid _WTS_LockState

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "nvda.pyw", line 394, in <module>
  File "core.pyc", line 702, in main
  File "core.pyc", line 634, in __init__
  File "winAPI\sessionTracking.pyc", line 288, in register
  File "winAPI\sessionTracking.pyc", line 205, in _setInitialWindowLockState
  File "winAPI\sessionTracking.pyc", line 220, in _isWindowsLocked_checkViaSessionQuery
  File "winAPI\sessionTracking.pyc", line 448, in _getSessionLockedValue
  File "enum.pyc", line 315, in __call__
  File "monkeyPatches\enumPatches.pyc", line 44, in _replacement__new__
  File "monkeyPatches\enumPatches.pyc", line 41, in _replacement__new__
  File "enum.pyc", line 582, in _missing_
ValueError: -1 is not a valid _WTS_LockState
INFO - winAPI.sessionTracking.unregister (13:36:50.391) - MainThread (2316):
Not unregistered session tracking, it was not registered.
```

### Expected behavior:

Install should complete normally.

### NVDA logs, crash dumps and other attachments:

[Install_NVDA.log](https://github.com/nvaccess/nvda/files/10026521/Install_NVDA.log)

#### NVDA version:
2022.3.2
#### Windows version:

Windows version: Windows 10 21H2 (10.0.19044) server

**Comment by josephsl** [comment]
Hi,

Looks like this user is trying to install NVDA in a Windows 10 Enterprise multi-session environment. I don't know if we officially support this.

Thanks.

**Comment by ThwinAg** [comment]
Hi
I also have the same issue when I push it out to windows 10 machines via SCCM. it's machine deployment and use script to install it silently. I can install it successfully if I log into machine and run the same script under admin account or system account. Nvda seems to check wts lock state since v2022.3.1 and I have been having the same issue with installing nvda via sccm since v2022.3.1.

**Comment by PhillipBrickner** [comment]
I also started to have issues installing from command line from batch file setupcomplete.cmd after imaging.  This had worked previously.  I am installing on Windows 10 Enterprise LTSC 21H2.  Install if ran from command line when logged in.

INFO - __main__ (11:04:36.589) - MainThread (3540):
Starting NVDA version 2022.3.2
INFO - core.main (11:04:36.714) - MainThread (3540):
Config dir: C:\Windows\TEMP\nsh3969.tmp\app\userConfig
INFO - config.ConfigManager._loadConfig (11:04:36.714) - MainThread (3540):
Loading config: C:\Windows\TEMP\nsh3969.tmp\app\userConfig\nvda.ini
WARNING - nvwave.WavePlayer.open (11:04:36.823) - MainThread (3540):
Unable to open WAVE_MAPPER device, there may be no audio devices.
INFO - core.main (11:04:36.854) - MainThread (3540):
Windows version: Windows 10 21H2 (10.0.19044) workstation
INFO - core.main (11:04:36.854) - MainThread (3540):
Using Python version 3.7.9 (tags/v3.7.9:13c94747c7, Aug 17 2020, 18:01:55) [MSC v.1900 32 bit (Intel)]
INFO - core.main (11:04:36.854) - MainThread (3540):
Using comtypes version 1.1.11
INFO - core.main (11:04:36.854) - MainThread (3540):
Using configobj version 5.1.0 with validate version 1.0.1
WARNING - nvwave.WavePlayer.open (11:04:38.776) - MainThread (3540):
Unable to open WAVE_MAPPER device, there may be no audio devices.
WARNING - tones.initialize (11:04:38.776) - MainThread (3540):
Failed to initialize audio for tones
Traceback (most recent call last):
  File "tones.pyc", line 28, in initialize
  File "nvwave.pyc", line 215, in __init__
  File "nvwave.pyc", line 301, in open
  File "nvwave.pyc", line 111, in _winmm_errcheck
FileNotFoundError: [Errno 2] A device ID has been used that is out of range for your system.
INFO - NVDAHelperLocal (11:04:38.870) - MainThread (3540):
Thread 3540, build\x86\localWin10\oneCoreSpeech.cpp, ocSpeech_initialize, 215:
ocSpeech_initialize

INFO - NVDAHelperLocal (11:04:38.870) - MainThread (3540):
Thread 3540, build\x86\localWin10\oneCoreSpeech.cpp, OcSpeechState::activate, 89:
Activating

INFO - NVDAHelperLocal (11:04:43.792) - MainThread (3540):
Thread 3540, build\x86\localWin10\oneCoreSpeech.cpp, preventEndUtteranceSilence_, 443:
AppendedSilence supported

INFO - synthDriverHandler.setSynth (11:04:43.808) - MainThread (3540):
Loaded synthDriver oneCore
WARNING - mathPres.initialize (11:04:43.854) - MainThread (3540):
MathPlayer 4 not available
INFO - core.main (11:04:44.042) - MainThread (3540):
Using wx version 4.1.1 msw (phoenix) wxWidgets 3.1.5 with six version 1.16.0
INFO - brailleInput.initialize (11:04:44.042) - MainThread (3540):
Braille input initialized
INFO - braille.initialize (11:04:44.042) - MainThread (3540):
Using liblouis version 3.22.0
INFO - braille.initialize (11:04:44.058) - MainThread (3540):
Using pySerial version 3.5
INFO - braille.BrailleHandler.setDisplayByName (11:04:44.183) - MainThread (3540):
Loaded braille display driver noBraille, current display has 0 cells.
CRITICAL - __main__ (11:04:44.339) - MainThread (3540):
core failure
ValueError: -1 is not a valid _WTS_LockState

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "nvda.pyw", line 394, in <module>
  File "core.pyc", line 702, in main
  File "core.pyc", line 634, in __init__
  File "winAPI\sessionTracking.pyc", line 288, in register
  File "winAPI\sessionTracking.pyc", line 205, in _setInitialWindowLockState
  File "winAPI\sessionTracking.pyc", line 220, in _isWindowsLocked_checkViaSessionQuery
  File "winAPI\sessionTracking.pyc", line 448, in _getSessionLockedValue
  File "enum.pyc", line 315, in __call__
  File "monkeyPatches\enumPatches.pyc", line 44, in _replacement__new__
  File "monkeyPatches\enumPatches.pyc", line 41, in _replacement__new__
  File "enum.pyc", line 582, in _missing_
ValueError: -1 is not a valid _WTS_LockState
INFO - winAPI.sessionTracking.unregister (11:04:44.354) - MainThread (3540):
Not unregistered session tracking, it was not registered.

**Comment by PhillipBrickner** [comment]
It looks like the last working version was 2022.2.4

**Comment by seanbudd** [comment]
Hi @PhillipBrickner , @ThwinAg 

We believe we have a fix ready for this issue, prepared in #14416.
Can you confirm this version of NVDA installs correctly?
https://ci.appveyor.com/api/buildjobs/b3xoqu8ow2boyqt3/artifacts/output%2Fnvda_snapshot_try-new-lock-tracking-27297%2C0b305bc4.exe

**Comment by seanbudd** [comment]
@PhillipBrickner @ThwinAg this should now be fixed on alpha as of [alpha-27325,304ea7a3](https://www.nvaccess.org/files/nvda/snapshots/alpha/nvda_snapshot_alpha-27325,304ea7a3.exe).

We will release this into 2022.3.3 and 2022.4beta4

**Comment by PhillipBrickner** [comment]
Yep it installed correctly.  Thanks!
